collector/settings.mdx

```diff
@@ -133,7 +126,7 @@ helper functions are expected to be defined, and the one we connect to for serve
     </tr>
     <tr>
       <td>db_password (<code>DB_PASSWORD</code>)</td>
-      <td>[none]</td>
+      <td>n/a, either this or db_url is required</td>
```

Comment:
I'm not sure that its correct that a password is required - AFAIK we can successfully connect over a unix domain socket with peer authentication if that's set up (or a TCP-based connection with trust, e.g. when restricting to local connections).

Reply:
It's not, but it's required in typical situations. I felt that this would be clearer. But I don't feel strongly about it; I'll drop this change.

Reply:
Yeah, makes sense - I do think its better to document how the code behaves here (i.e. it doesn't require it), and if needed we could add additional commentary as to what's typical (but I think its okay if we omit here, since I think most people understand they'd be using a password).